<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>track_dialog</class>
 <widget class="QDialog" name="track_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>269</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Track Orientation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="reference_threshold_label">
     <property name="text">
      <string>Reference Threshold:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="reference_threshold">
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScientificDoubleSpinBox" name="angular_tolerance">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.070000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="conserve_memory">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, a different version of the internal function will be utilized that uses less RAM, but also runs a little slower.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Conserve Memory</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="angular_limit_label">
     <property name="text">
      <string>Angular Limit:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="resolution_limit_label">
     <property name="text">
      <string>Resolution Limit:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="angular_tolerance_label">
     <property name="text">
      <string>Angular Tolerance:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ScientificDoubleSpinBox" name="angular_limit">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="value">
      <double>29.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ScientificDoubleSpinBox" name="resolution_limit">
     <property name="suffix">
      <string> Å</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.400000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="crystal_id_label">
     <property name="text">
      <string>Starting Crystal ID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="crystal_id">
     <property name="maximum">
      <number>10000000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.py</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>crystal_id</tabstop>
  <tabstop>angular_tolerance</tabstop>
  <tabstop>angular_limit</tabstop>
  <tabstop>resolution_limit</tabstop>
  <tabstop>reference_threshold</tabstop>
  <tabstop>conserve_memory</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>track_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
